content/manuals/desktop/features/model-runner.md

```diff
@@ -17,10 +17,15 @@ The Docker Model Runner plugin lets you:
 - [Pull models from Docker Hub](https://hub.docker.com/u/ai)
 - Run AI models directly from the command line
 - Manage local models (add, list, remove)
-- Interact with models using a submitted prompt or in chat mode
+- Interact with models using a submitted prompt or in chat mode in the CLI or Docker Desktop Dashboard
+- Push models to Docker Hub
 
 Models are pulled from Docker Hub the first time they're used and stored locally. They're loaded into memory only at runtime when a request is made, and unloaded when not in use to optimize resources. Since models can be large, the initial pull may take some time — but after that, they're cached locally for faster access. You can interact with the model using [OpenAI-compatible APIs](#what-api-endpoints-are-available).
 
+> [!TIP]
+>
+> Using Testcontainers? [Testcontainers for Java](https://java.testcontainers.org/modules/docker_model_runner/) and [Go](https://golang.testcontainers.org/modules/dockermodelrunner/) now support Docker Model Runner.
```

Comment:
Should we similarly link out to Docker Compose here, once https://github.com/docker/docs/pull/22392 is merged?